.github/workflows/node.js.yml

```
@@ -32,16 +32,27 @@ jobs:
               uses: coactions/setup-xvfb@v1
               with:
                   run: npm test
-            - name: Code coverage
+            - name: Toolkit code coverage
```

should we use the same naming pattern as the existing `Code coverage (CodeWhisperer)` item, or change that one? it's probably visually helpful to have a common prefix

```suggestion
            - name: Code coverage (Toolkit)
```